docs/stimuli.rst

```diff
@@ -0,0 +1,44 @@
+Stimuli Types
+=============
+
+Higher-level analyses available in ``ipfx`` are critically-dependent on stimulus type and naming conventions.
+Unless overwritten, all :py:class:`~ipfx.dataset.ephys_data_set.EphysDataSet` instances create a default :py:class:`~ipfx.stimulus.StimulusOntology` based on
+`stimulus_ontology.json <https://github.com/AllenInstitute/ipfx/blob/master/ipfx/defaults/stimulus_ontology.json>`_.
+The job of this class is to provide mechanisms for searching for stimuli that have been tagged with standardized names.
```

Comment:
"Instances of this class provide ..."

Might also want to describe the context of this search (when and why, what dataset is being searched). This could go in an overall description of the stimulus ontology system at the top.

Reply:
I have added more details, thanks.